### PR TITLE
feat(out-of-support): make the message clearer

### PR DESCRIPTION
**Pull request by tbouffard**
Also remove the link to the downloads page are it doesn't relate to the documentation but to the
products themselves.

Here, we want to warn the reader about an outdated documentation content, not about its usage of
an out-of support product.

### Proposals

previous implementation | 179a073 | e09219b
---- | ---- | ----
previous |  proposition_01 | proposition_02

### Inspirations

Inspired by the Posgres documentation: https://www.postgresql.org/docs/11/app-createdb.html

image

### Reviews

I'm waiting for the feedback I've requested from people who use out of support content or who are in contact with customers using such content:
- @ugaston

ℹ️ Test page: https://bonitasoft-bonita-documentation-theme-build_preview-pr-234.surge.sh/msg-block-out-of-support.html

### CODE REVIEW

**Review by ugaston — COMMENTED** [review]
Since we are also updating the link so it points to the latest version of the same page (instead of pointing to the home page of the latest version), I would have a slight preference for the other wording you suggested, because I feels more straightforward and less ambiguous, that is:

"You may want to view the same page for the latest version instead."

Furthermore, I would go all the way in copycatting PostgreSQL documentation, and also add a little message letting people know they can choose to view any other version of the same page by clicking on the corresponding link in the drop-down menu at the top-right corner. Something like: 

"You may want to view the same page for the latest version instead, or for any of the versions available in the drop-down menu at the top-right corner."

**Comment by tbouffard** [comment]
I will integrate @ugaston feedback soon.

File: src/partials/toolbar.hbs
Comment (tbouffard) [reply src/partials/toolbar.hbs:44]: Fixed in 911d1c6c

**Comment by tbouffard** [comment]
## To review @ugaston proposal

Commit: 721d94a

### Large screens

image


### Small screens prior switching to mobile view

⚠️ may conflict with changes proposed in #233 

image

**Review by ugaston — APPROVED** [review]
Looks good to me!

Thank you, Thomas!

**Comment by tbouffard** [comment]
Perfect 
I am waiting for the review of a maintainer of the repo to be able to merge this.

**Review by benjaminParisel — APPROVED** [review]

**Pull request merged**
